src/Configuration/ConfigurationFactory.php

```diff
@@ -327,6 +327,7 @@ private function retrieveLogs(Logs $logs, string $configDir, ?bool $useGitHubLog
             self::pathToAbsolute($logs->getHtmlLogFilePath(), $configDir),
             self::pathToAbsolute($logs->getSummaryLogFilePath(), $configDir),
             self::pathToAbsolute($logs->getJsonLogFilePath(), $configDir),
+            self::pathToAbsolute($logs->getGitlabLogFilePath(), $configDir),
```

Comment:
I think we also need `--logger-gitlab` or something like this, to 

- make it consistent with `--logger-github`
- to have option and config file parity, where this feature can be enabled either from option (`--logger-gitlab`) or from config file by setting `"github": "/path/to/file.json"`

Related to https://github.com/infection/infection/issues/141